Source/WebCore/Modules/modern-media-controls/controls/text-tracks.css

```
@@ -45,7 +45,7 @@ video::-webkit-media-text-track-container {
     -webkit-line-box-contain: block inline-box replaced;
 }
 
-video[controls]::-webkit-media-text-track-container.visible-controls-bar {
+video[controls][pseudo="-webkit-media-text-track-container.visible-controls-bar"].visible-controls-bar {
```

```suggestion
video[controls][pseudo="-webkit-media-text-track-container"].visible-controls-bar {
```

It's invalid CSS syntax otherwise

---

Done in [3fb18c8](https://github.com/WebKit/WebKit/commit/3fb18c8f7d1167791a66c1904ffbf1a107168d4d).